carrr-client/src/components/NavBar.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <b-navbar type="dark" variant="dark">
+    <b-container>
+      <router-link to="/" tag="b-navbar-brand">CarRR</router-link>
```

Comment:
We should put the full app name here. (?)

Reply:
It should be logo